asciidoc/cypher-workflow.adoc

```diff
@@ -274,6 +274,58 @@ Some administrative operations must be carried out against the `system` database
 To do this, construct a session targeting the database called `system` and execute Cypher as usual.
 Please see <<cypher-manual#administration-databases-introduction, Cypher Manual -> Databases>> for more information.
 
+[role=enterprise-edition]
+[[database-selection]]
+=== Database selection
+
+The database selection on the client side happens on the <<driver-session-api>>.
+You pass the name of the database to the driver during session creation.
+The default database is used if you don't specify a name.
+A database name must not be literal `null` nor an empty string.
+
+NOTE: While there are no different versions of the driver for the community or enterprise edition of Neo4j, those methods
+      will only work when the driver is connected against the enterprise edition of Neo4j.
```

Comment:
```suggestion
NOTE: Selection of database is only available when the driver is connected against Enterprise Edition of Neo4j.
```

Reply:
I have changed your suggestion a bit… The API is still available in the driver, but it will throw an error against community edition. I think that bit is important to have.

Reply:
I approve! Looks good, well done!